tests/acceptance/apps/sidebar-test.js

```diff
@@ -56,7 +56,7 @@ test('includes breadcrumb to parent stack', function(assert) {
   signInAndVisit(`/apps/${appId}`);
 
   andThen(function() {
-    let stackHandle = find(`h3:contains(${stackHandle})`);
+    let stackHandle = find(`h3`); // TODO this should read :contains(somehandle)
```

Comment:
This shenanigans is fixed in the private repo:
https://github.com/aptible/dashboard-private/pull/1/files#diff-19986cf1b40b83693b1286647a6830eaR8